contributors/devel/sig-instrumentation/migration-to-structured-logging.md

```diff
@@ -7,6 +7,83 @@ should be applicable at later stages or for other projects using `klog` logging
 
 [Structured Logging KEP]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging
 
+## How to contribute
+
+### About the migration
+
+We would like for the Kubernetes community to settle on one preferred log message structure, that will be enforced by new klog methods. The goal of the migration is to switch C like format string logs to structured logs with explicit metadata about parameters.
+
+Migration within the structured logging working group happens in two different ways - organized & non-organized. 
+
+With organized, milestone-based, large-scale migration we try to target deliverables for a specific Kubernetes release. A good example of such an effort is the migration of the entire Kubelet code in the [#1.21 release](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.21.md#structured-logging-in-kubelet) 
+
+For non-organized migrations, the onus is, generally, on the individual contributors.
+
+* Organized Migration
+
+Organized migration is carried out in a two state cycle aligning with the cadence of Kubernetes releases. 
+
+In the first stage, we pick a particular migration milestone & create an issue to split the work into smaller chunks (for example, migrating just a single file). This ensures that the work can be easily picked up by multiple individual contributors and also avoids conflicts. After the migration activity is complete, we mark the directory as migrated to avoid regressions with the help of tooling that we have developed. 
+
+In the second milestone, we take a break from migration to analyze results & improve our existing processes. Adding structured information to logs is a very costly & time-consuming affair. Setting aside time in the second stage to collect feedback, analyze the impact of changes on the log volume & performance, and better our PR review process helps us avoid mistakes and duplicate efforts.
+
+* Non-organized migration
+
+As aforementioned, our non-organized migration efforts are spearheaded by individual contributors who require particular code sections to utilize new features early. 
```

Comment:
Not quite sure what's meant by this - can you elaborate?

Reply:
I think the meaning here is that
can migrate separately when there is no migirage plan for the particular code sections but want to utilize new features early.

Reply:
I'd re-word this as follows:
```suggestion
As aforementioned, our non-organized migration efforts are spearheaded by individual contributors who need to migrate particular code sections to utilize new features early. 
```